.drone.yml

```diff
@@ -226,7 +226,23 @@ pipeline:
     when:
       repo: vmware/vic
       event: push
-      branch: [master, 'releases/*']
+      branch: [master]
+      status: success
+
+  publish-gcs-builds-on-release-push-pass:
+    image: 'victest/drone-gcs:1'
+    pull: true
+    secrets:
+      - google_key
+    source: bundle
+    target: vic-engine-builds/${DRONE_BRANCH}
```

Comment:
The `jenkins-nightly-run.sh` script references these builds for download. I suggest we update it to allow specifying the branch of the vic bundle as well as the build number. It doesn't currently support a subfolder here at all, so some change is needed.

Reply:
Updated

Reply:
It depends how drone implements it, but the gsutil semantics suggest a `/` on the end is a useful precaution.